### ISSUE TITLE
Translation of Linktext duplicates the Output of the link Label

### ISSUE THREAD

**Issue by anderlk** [tritum/form_element_linked_checkbox#24]
Translate the Linktext in xlf with

`buchungsformular_1.element.linkedcheckbox-1.properties.linkText`

The Output Doubles the linked word

Output:
 I have read the **Privacy policy** and agree to the processing of my data. **Privacy policy** *

**Comment by wazum** [comment]
I can confirm this.

The template `form_element_linked_checkbox/Resources/Private/Frontend/Partials/LinkedCheckbox.html` is wrong, this change fixes it (remove the typolink part):

```
14:                <span>{formvh:translateElementProperty(element: element, property: 'label') -> f:format.raw()}<f:if condition="{element.required}"> <f:render partial="Field/Required" /></f:if></span>
```

**Comment by anderlk** [comment]
Thanks, ist works perfect.

**Comment by iamwebrocker** [comment]
Hi,

whats the status of this? According to the reply in the closed #25, this behaviour is intended.
But if I use %s in the label - which btw is the main reason to use this element - I get the duplicated output.
Now I wanted to override the code in the partial, but the author uses an absurdly high key for this in two places, so it is not possible for me to override the partial path in the Form-Setting without going even absurdly higher with my definition.

in `ext/form_element_linked_checkbox/Configuration/Yaml/FormSetup.yaml`:
```
TYPO3:
  CMS:
    Form:
      prototypes:
        standard:
          formElementsDefinition:
            Form:
              renderingOptions:
                partialRootPaths:
                  1505042806: 'EXT:form_element_linked_checkbox/Resources/Private/Frontend/Partials/'
```

in `ext/form_element_linked_checkbox/Configuration/TypoScript/setup.typoscript`:
```
plugin.tx_form {
    settings {
        yamlConfigurations {
            1505042806 = EXT:form_element_linked_checkbox/Configuration/Yaml/FormSetup.yaml
        }
    }
}
```
Am I missing an obvious path here? How do I get rid of the duplicated linkText?

Anyway - thanks for the extension, this form element is really easing the pain with those Data Processing Accept thingies

**Comment by eliashaeussler** [comment]
Hi together :wave:

Can you please try to apply the following patch and check if that solves the problem? I'd provide a PR then.

```diff
diff --git a/Classes/Hooks/FormElementLinkResolverHook.php b/Classes/Hooks/FormElementLinkResolverHook.php
index b925e39..52811a1 100644
--- a/Classes/Hooks/FormElementLinkResolverHook.php
+++ b/Classes/Hooks/FormElementLinkResolverHook.php
@@ -95,6 +95,7 @@ class FormElementLinkResolverHook
         $renderable->setProperty('_label', $label);
         $renderable->setProperty('_linkText', $translatedLinkText);
         $renderable->setProperty('_pageUid', $pageUid);
+        $renderable->setProperty('_linksProcessed', true);
     }
 
     /**
@@ -174,4 +175,4 @@ class FormElementLinkResolverHook
         } while(strpos($filteredValue, '%%') !== false);
         return strpos($filteredValue, '%') !== false;
     }
-}
\ No newline at end of file
+}
diff --git a/Resources/Private/Frontend/Partials/LinkedCheckbox.html b/Resources/Private/Frontend/Partials/LinkedCheckbox.html
index 9ed645a..6a47252 100644
--- a/Resources/Private/Frontend/Partials/LinkedCheckbox.html
+++ b/Resources/Private/Frontend/Partials/LinkedCheckbox.html
@@ -11,9 +11,9 @@
                         errorClass="{element.properties.elementErrorClassAttribute}"
                         additionalAttributes="{formvh:translateElementProperty(element: element, property: 'fluidAdditionalAttributes')}"
                 />
-                <span>{formvh:translateElementProperty(element: element, property: 'label') -> f:format.raw()} <f:link.typolink parameter="{element.properties.pageUid}" target="_blank">{formvh:translateElementProperty(element: element, property: 'linkText')}</f:link.typolink><f:if condition="{element.required}"> <f:render partial="Field/Required" /></f:if></span>
+                <span>{formvh:translateElementProperty(element: element, property: 'label') -> f:format.raw()} <f:if condition="!{element.properties._linksProcessed}"><f:link.typolink parameter="{element.properties.pageUid}" target="_blank">{formvh:translateElementProperty(element: element, property: 'linkText')}</f:link.typolink></f:if><f:if condition="{element.required}"> <f:render partial="Field/Required" /></f:if></span>
             </label>
         </div>
     </f:render>
 </formvh:renderRenderable>
-</html>
\ No newline at end of file
+</html>
```
